anesthetic/samples.py

```diff
@@ -300,6 +308,14 @@ def plot_2d(self, axes, *args, **kwargs):
                     pos = ax.position
                     lkwargs = local_kwargs.get(pos, {})
                     lkwargs['kind'] = kind.get(pos, None)
+                    # TODO: remove this in version >= 2.1
+                    if lkwargs['kind'] == 'kde':
+                        warnings.warn(
+                            "You are using \'kde\' as a plot kind. "
+                            "\'kde_1d\' and \'kde_2d\' are the appropriate "
+                            "keywords for anesthetic. Your plots may look "
+                            "odd if you use this argument."
+                            )
```

Comment:
I believe the same should go for `lkwargs['kind'] == 'hist'`. So probably good to add that here, too.

Reply:
done in 549979c